tests/atoms/gaia_apps.js

```diff
@@ -373,12 +373,14 @@ var GaiaApps = {
     }
 
     let origin = app.origin;
-    console.log('app with origin \'' + origin + '\' is displayed');
+    let manifest_url = app.manifest_url;
+    console.log('app with origin and manifest url \'' + origin + manifest_url + '\' is displayed');
```

Comment:
I think I would remove the origin part here.
And I would remove the variable declarations of origin here and just use app.origin and app.manifest_url directly.